lua/weapons/weapon_autism/shared.lua

```diff
@@ -19,7 +19,11 @@ function SWEP:PrimaryAttack()
 		self:ExtEmitSound(soundfile, {speech=-1, pitch=pit})
 		local stime = SoundDuration(soundfile)*100/pit
 		self.Owner:SendLua("util.ScreenShake(LocalPlayer():GetPos(), 5, 0.5, "..tostring((stime*1.65))..", 10000 )")
-		timer.Simple(math.Rand(0,1), function() setPlayerGesture(self.Owner, GESTURE_SLOT_ATTACK_AND_RELOAD, table.Random(autisticTwitches), true ) end)
+		if IsValid(self) and IsValid(self.Owner) then
+			timer.Simple(math.Rand(0,1), function()

```

Comment:
This doesn't fix it, you need to check isvalid inside the timer. The issue happens because the player dies between the timer creation and the function call